docs/reference-guide/storage-modes.md

```diff
@@ -26,6 +32,17 @@ If you are running the Memgraph Enterprise Edition, you need to have
 [`STORAGE_MODE` permission](/reference-guide/auth-module.md) to change the
 storage mode.
 
+Switching from the in-memory storage mode to disk mode is allowed only when
+the database is empty and there is a single session active. The intention here
+is to allow switching to users only upon starting Memgraph, not later. Switching from
+disk mode to in-memory is forbidden since data may not fit into the memory.
+
+To switch to disk storage mode the following query can be used:
+
+```cypher
+STORAGE MODE ON_DISK_TRANSACTIONAL;
+```
```

Comment:
```suggestion
Switching from the in-memory storage mode to the on-disk storage mode is allowed when there is only one active session and the database is empty. As Memgraph Lab uses multiple sessions to run queries in parallel, it is currently impossible to switch to the on-disk storage mode within Memgraph Lab. You can change the storage mode to on-disk transactional using `mgconsole`, then connect to the instance with Memgraph Lab and query the instance as usual.

When querying the database, all the graph objects used in a transaction need to be able to fit in the available RAM, otherwise the database will raise an exception. 

To change the storage mode to `ON_DISK_TRANSACTIONAL`, use the following query:

```cypher
STORAGE MODE ON_DISK_TRANSACTIONAL;
```

Changing the storage mode from on-disk to in-memory is forbidden since data may not fit into the memory.

Reply:
@as51340  be carfeul when merging this, i don't know why it doesn't recognize the last ``` i used to close the suggestion tag